### DISCUSSION TITLE
System.IO.MemoryStream support for MediaElement

### DISCUSSION THREAD

**Discussion by rozgonyiadam** [CommunityToolkit/Maui#1384] [New Feature Discussions]
I would like to propose a [similar feature](https://github.com/dotnet/maui/issues/15432). I should be able to bind a MemoryStream object to the MediaElement's Source (currently, it is only possible with MediaSource). The reason for this is that I am receiving a stream from the web that also needs writing during playback because of chunks. This requires support for seamless playback during stream buffering.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
